scene/gui/code_edit.h

```diff
@@ -332,6 +334,9 @@ class CodeEdit : public TextEdit {
 	void set_auto_indent_enabled(bool p_enabled);
 	bool is_auto_indent_enabled() const;
 
+	void set_tab_always_indents(const bool p_tab_always_indents);
```

Comment:
```suggestion
	void set_tab_always_indents(bool p_tab_always_indents);
```
No `const` for simple types

Reply:
Also the wording of this needs to be discussed, using the `is_` syntax without `_enabled` makes things a bit confusing